modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/TxManagerImpl.java

```diff
@@ -192,6 +196,11 @@ public class TxManagerImpl implements TxManager, NetworkMessageHandler {
 
     private final ReplicaService replicaService;
 
+    /** Registry of locally started active transactions. */
+    private final Map<UUID, InternalTransaction> transactions = new ConcurrentHashMap<>();
+
+    private final TransactionsViewProvider txSystemViewProvider = new TransactionsViewProvider(transactions);
```

Comment:
do we really need `viewProvide` as a member of `TxManager`? The former is stateless object with no particular lifecycle. Besides, system views are supposed to be registered only once. Therefore, it would be better I think to not pollute manager with unnecessary members (e.g. create provider and immediately call invoke `get()` method inside `systemViews()` method).

Reply:
Thanks, the PR was reworked so that the system view now uses existing collections. But now provider requires data that is become available only after component start